pyproject.toml

```diff
@@ -22,7 +22,7 @@ dependencies = [
     "appdirs",
     "lightning>=2.0",
     "click",
-    "depthcharge-ms>=0.2.3,<0.3.0",
+    "depthcharge-ms>=0.4.8 ",
```

Comment:
**todo:** Let's add a `<0.5.0`.

I don't have breaking changes planned in depthcharge, but its easier to make it more liberal if it becomes unnecessary.

Reply:
Makes sense. Done!